libraries/ms-drivers/inc/adt_7470_fan_controller.h

```diff
@@ -0,0 +1,39 @@
+#pragma once
+
+#include "adc.h"
+#include "gpio.h"
+#include "i2c.h"
+#include "soft_timer.h"
+
+typedef void (*Adt7470DataCallback)();
```

Comment:
wouldn't hurt to include a `void *context` parameter, makes it simpler to use externally